<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapCreator</class>
 <widget class="QMainWindow" name="MapCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map Creator</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="goButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>70</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QLabel" name="widthLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>46</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Width:</string>
    </property>
   </widget>
   <widget class="QLabel" name="heightLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>46</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Height:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>921</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="sizeLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; text-decoration: underline;&quot;&gt;Map Size&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>621</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QSpinBox" name="wValue">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Min = 5, Max = 30</string>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="hValue">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>70</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Min = 5, Max = 30</string>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>110</y>
      <width>20</width>
      <height>511</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="wallGroup">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>130</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Walls</string>
    </property>
    <widget class="ClickLabel" name="wallImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/images/wall.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="startGroup">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>130</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Start</string>
    </property>
    <widget class="ClickLabel" name="startImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/images/startdoor.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="endGroup">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>130</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>End</string>
    </property>
    <widget class="ClickLabel" name="endImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/images/enddoor.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="chestGroup">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>230</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Chests</string>
    </property>
    <widget class="ClickLabel" name="chestImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/images/chest.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="monsterGroup">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>320</y>
      <width>221</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Monsters</string>
    </property>
    <widget class="ClickLabel" name="ratImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>46</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/images/rat.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="emptyGroup">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>230</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Floor</string>
    </property>
    <widget class="ClickLabel" name="emptyImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/images/mud.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="mapText">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; font-style:italic;&quot;&gt;Map Editor&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; font-style:italic; color:#ff0000;&quot;&gt;Dungeon Crawler!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>922</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="separator"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Help"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="text">
    <string>&amp;Help Dialog</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickLabel</class>
   <extends>QLabel</extends>
   <header>clicklabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>goButton</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
